addons/sale/models/account_move.py

```diff
@@ -185,13 +185,14 @@ def _get_partner_credit_warning_exclude_amount(self):
             exclude_amount += order_amount_company
         return exclude_amount
 
+    # todo need to remove both the field and compute method in master as this field is neither used in python nor in XML
     @api.depends('line_ids.sale_line_ids.order_id', 'currency_id', 'tax_totals', 'date')
     def _compute_partner_credit(self):
         super()._compute_partner_credit()
         for move in self.filtered(lambda m: m.is_invoice(include_receipts=True)):
             sale_orders = move.line_ids.sale_line_ids.order_id
             amount_total_currency = move.currency_id._convert(
-                move.tax_totals['amount_total'],
+                move.tax_totals['total_amount'],
                 move.company_currency_id,
                 move.company_id,
                 move.date
```

Comment:
```suggestion
                amount_total_currency = move.tax_totals['total_amount_currency']
```
How about something like this? It seems more correct, since the tax_totals is already handling all the rounding issues, and might be better than doing an additional conversion.